runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -481,6 +623,11 @@ I_32
 MM_StandardAccessBarrier::backwardReferenceArrayCopyIndex(J9VMThread *vmThread, J9IndexableObject *srcObject, J9IndexableObject *destObject, I_32 srcIndex, I_32 destIndex, I_32 lengthInSlots)
 {
 	I_32 retValue = ARRAY_COPY_NOT_DONE;
+	//TODO Evgenia SATBBarrier re-enable opt?
+	J9JavaVM *javaVM = vmThread->javaVM;
+	if (javaVM->gcWriteBarrierType == gc_modron_wrtbar_realtime) {
+		return retValue;
+	}
```

Comment:
note to other reviewers: I'm ok with this hack for now (it forces slow path array copying).